spec/lib/link_checker_spec.rb

```diff
@@ -142,6 +142,26 @@
       include_examples "has warnings"
     end
 
+    context "header with CR/LF character" do
+      let(:uri) { "http://www.not-gov.uk/header_with_CRLF_character" }
+      before do
+        stub_request(:get, uri)
+          .to_raise(ArgumentError.new("header field value cannot include CR/LF"))
```

Comment:
I think we can stub this in a somewhat more realistic way with:

```
        stub_request(:get, uri)
          .to_return(headers: { "Invalid" => "With a carriage return \r character" })
```

This could be a bit more useful as we'd learn if a dependency change broke the fix.